content/en/docs/tasks/run-application/configure-pdb.md

```diff
@@ -135,31 +135,11 @@ You can find examples of pod disruption budgets defined below. They match pods w
 
 Example PDB Using minAvailable:
 
-```yaml
-apiVersion: policy/v1beta1
-kind: PodDisruptionBudget
-metadata:
-  name: zk-pdb
-spec:
-  minAvailable: 2
-  selector:
-    matchLabels:
-      app: zookeeper
-```
+{{< codenew file="policy/zookeeper-pod-disruption-budget-minavailable.yaml" >}}
```

Comment:
@sftim while i love the fact that you used the new shortcode, could you also add a note below or a sentence to say s'thing like _You can create it using `kubectl create -f https://`_ so folks new to docs can just copy the single line and off you go.

Of course those who are familiar with the docs can easily copy the url address from the block itself

Reply:
of course, this assumes the example is already present in the examples dir, if not i'm as part of this PR we should add it